.github/workflows/test-ci-windows.yml

```diff
@@ -0,0 +1,333 @@
+        Write-Host "IIS modules: $modules"
+        if ($LASTEXITCODE -ne 0) {
+            Write-Error "appcmd failed with exit code $LASTEXITCODE"
+            exit 1
+        }
+
+        if (-not ($modules -match "ModSecurity")) {
+            Write-Error "ModSecurity module not found in IIS modules"
+            Write-Host "IIS modules: $modules"
+            exit 1
+        }
+
+        $testCases = @(
+            @{Url = "http://localhost/"; Description = "Normal request"; ExpectedCode = 200},
+            @{Url = "http://localhost/?id=1' OR '1'='1"; Description = "SQL injection attempt"; ExpectedCode = 403},
+            @{Url = "http://localhost/?q=<script>alert('test')</script>"; Description = "XSS attempt"; ExpectedCode = 403}
+        )
+        
+        foreach ($test in $testCases) {
+            try {
+                $response = Invoke-WebRequest $test.Url -UseBasicParsing -SkipHttpErrorCheck -TimeoutSec 30
+                
+                if ($response.StatusCode -eq $test.ExpectedCode) {
+                    Write-Host "PASS: $($test.Description) - returned $($response.StatusCode)"
+                }
+                else {
+                    Write-Host "FAIL: $($test.Description) - expected $($test.ExpectedCode) but got $($response.StatusCode)"
+                }
+            }
+            catch {
+                Write-Host "ERROR: $($test.Description) - request failed: $($_.Exception.Message)"
+            }
+        }
+        
+
+        # Check event log
+        $badMessagePattern = 'Failed to find the RegisterModule entrypoint|The description for Event ID|The data is the error|dll failed to load'
+
+        $events = Get-EventLog -LogName Application -Newest 100 |
+            Where-Object { $_.Message -match $badMessagePattern } |
+            Where-Object { $_.Source -match 'IIS|W3SVC|mscor|IIS-W3SVC|IIS-W3WP|ModSecurity' }
+
+        if ($events -and $events.Count -gt 0) {
+            Write-Host '::error:: Found errors in event log'
+            $events | Select-Object TimeGenerated, Source, EntryType, EventID, Message | Format-List
+            Exit 1
+        }
+
+        Get-EventLog -LogName Application -Source ModSecurity | Format-List
+
+    - name: Install go-ftw
+      shell: pwsh
+      run: |
+        go install github.com/coreruleset/go-ftw@latest
+
+    - name: Test ModSecurity Rules
+      shell: pwsh
+      run: |
+        $testRuleDir = "C:\Program Files\ModSecurity IIS\coreruleset\tests\regression\tests"
+        $goBinPath = ""
+        if ($env:GOBIN) {
+            $goBinPath = $env:GOBIN
+        } elseif ($env:GOPATH) {
+            $goBinPath = Join-Path $env:GOPATH "bin"
+        } else {
+            $goBinPath = Join-Path $env:USERPROFILE "go\bin"
+        }
+
+        & "$goBinPath\go-ftw.exe" run -d $testRuleDir --cloud -e "920100-2$|920100-4$|920100-8$|920100-12$|920272-5$|920290-1$|920620-1$|920380-1$" --show-failures-only
```

Comment:
This is awesome (I mean you added CRS as a test), but why does it need to exclude these rules?